tests/Livewire/InstallWizardTest.php

```diff
@@ -2,28 +2,182 @@
 
 namespace Tests\Feature\Livewire;
 
+use FluxErp\Livewire\Auth\Login;
 use FluxErp\Livewire\InstallWizard;
 use FluxErp\Tests\TestCase;
 use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Str;
 use Livewire\Livewire;
 
 class InstallWizardTest extends TestCase
 {
-    public function test_renders_successfully()
+    public function setUp(): void
```

Comment:
why even bother setting Config key in setup when it is set separately in every method except one?